pkgs/applications/misc/cherrytree/default.nix

```diff
@@ -20,17 +20,16 @@ pythonPackages.buildPythonApplication rec {
   meta = with lib; {
     description = "An hierarchical note taking application";
     longDescription = ''
-      Cherrytree is an hierarchical note taking application,
-      featuring rich text, syntax highlighting and powerful search
-      capabilities. It organizes all information in units called
-      "nodes", as in a tree, and can be very useful to store any piece
-      of information, from tables and links to pictures and even entire
-      documents. All those little bits of information you have scattered
-      around your hard drive can be conveniently placed into a
-      Cherrytree document where you can easily find it.
+      Cherrytree is an hierarchical note taking application, featuring rich
+      text, syntax highlighting and powerful search capabilities. It organizes
+      all information in units called "nodes", as in a tree, and can be very
+      useful to store any piece of information, from tables and links to
+      pictures and even entire documents. All those little bits of information
+      you have scattered around your hard drive can be conveniently placed into
+      a Cherrytree document where you can easily find it.
     '';
     homepage = "http://www.giuspen.com/cherrytree";
     license = licenses.gpl3;
-    maintainers = with maintainers; [ AndersonTorres ];
```

Comment:
Are you sure you want to remove yourself from this list? while you are still maintaining it ;) (i'm also wondering why)

Reply:
Well, now I am using Emacs org mode for those things :) and I don't see any reason to maintain or use it nowadays.

The fact I am the only maintainer of it is a bit sad, though.